### DISCUSSION TITLE
Descriptions plugins and more items

### DISCUSSION THREAD

**Discussion by felipeflorian** [LimberDuck/nessus-file-reader#9] [General]
Hi! I want to know if it is possible to get the description or see also attributes that are shown in Nessu. It would be very helpful.

**Comment by damian-krawczyk** [comment]
Hi @felipeflorian yes, it's possible. You can take **any** element from `ReportItem` (reported plugin result within nessus file), for `see also` and `description` you can do it as shown below:
```python
      see_also = nfr.plugin.report_item_value(report_item, 'see_also')
      description = nfr.plugin.report_item_value(report_item, 'description')
```
See this example:
https://github.com/LimberDuck/nessus-file-reader/blob/577e21cc298875e86d118051d56a288953c1dc94/examples/nfr-plugin-example.py#L1-L11
